flask_monitoringdashboard/__init__.py

```diff
@@ -29,12 +29,13 @@ def loc():
 blueprint = Blueprint('dashboard', __name__, template_folder=loc() + 'templates')
 
 
-def bind(app, schedule=True):
+def bind(app, schedule=True, micro=False):
```

Comment:
maybe we should name it `without_dashboard=False`?

Reply:
I can do that - I tried a couple different names but figured I should have it match with everything else being called micro.